x/apps/genesis.go

```diff
@@ -32,17 +34,20 @@ func InitGenesis(ctx sdk.Ctx, keeper keeper.Keeper, supplyKeeper types.AuthKeepe
 	// check if the staked pool accounts exists
 	stakedPool := keeper.GetStakedPool(ctx)
 	if stakedPool == nil {
-		panic(fmt.Sprintf("%s module account has not been set", types.StakedPoolName))
+		fmt.Println(fmt.Sprintf("%s module account has not been set", types.StakedPoolName))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("%s module account has not been set", types.StakedPoolName)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)